builders/interoperability/xcm/xc20/send-xc20s/xcm-pallet.md

```diff
@@ -0,0 +1,364 @@
+---
+title: Send XC-20s to Other Chains
+description: This guide introduces the PolkadotXCM Pallet and explains how to send XC-20s to another chain using some of the pallet's extrinsics.
+---
+
+# Using the PolkadotXCM Pallet To Send XC-20s
+
+## Introduction {: #introduction }
+
+!!! note
+
+    The PolkadotXCM Pallet replaces the deprecated XTokens Pallet. Accordingly, ensure that you are using the PolkadotXCM Pallet to interact with XC-20s.
+
+Manually crafting an XCM message for fungible asset transfers is a challenging task. Consequently, developers can leverage wrapper functions and pallets to use XCM features on Polkadot and Kusama. One example of such wrappers is the [XCM](https://wiki.polkadot.network/docs/learn-xcm-pallet){target=\_blank} Pallet, which provides different methods to transfer fungible assets via XCM.
+
+This guide will show you how to leverage the PolkadotXCM Pallet to send [XC-20s](/builders/interoperability/xcm/xc20/overview/){target=\_blank} from a Moonbeam-based network to other chains in the ecosystem (relay chain/parachains).
+
+**Developers must understand that sending incorrect XCM messages can result in the loss of funds.** Consequently, testing XCM features on a TestNet is essential before moving to a production environment.
+
+## Nomenclature {: #nomenclature }
+
+Because there are various XCM-related pallets and precompiles with similar-sounding names, the following section will clarify the differences between each. 
+
+- `PolkadotXCM` - this pallet (and the focus of this page) enables you to interact with XC-20s on Moonbeam, replacing the deprecated `XTokens` pallet 
+- `pallet-xcm`- the general Polkadot XCM pallet allows you to interact with cross-chain assets. Moonbeam's `PolkadotXCM` pallet is essentially a wrapper of `pallet-xcm`. Because of this, you may see `PolkadotXCM` and `pallet-xcm` referred to interchangeably
+- `XTokens` - This pallet is now deprecated and replaced by `PolkadotXCM`
+- `XCMInterface.sol` - This precompile is the solidity interface that replaces `XTokens.sol` and enables you to interact with the methods of `PolkadotXCM` from the EVM via a solidity interface
+
+## PolkadotXCM Pallet Interface {: #polkadotxcm-pallet-interface }
+
+### Extrinsics {: #extrinsics }
```

Comment:
While good, on Moonbeam/Moonriver, only these calls are allowed -> https://github.com/moonbeam-foundation/moonbeam/blob/master/runtime/moonbeam/src/lib.rs#L1204-L1209

Moonbase ALpha is a bit more permissive, but unsure what to do here because we might not want to show all extrinsic?